ssc/cardano-sl-ssc.cabal

```diff
@@ -65,6 +65,7 @@ library
                      , cardano-sl-crypto
                      , cardano-sl-db
                      , cardano-sl-lrc
+                     , cardano-sl-networking
```

Comment:
@erikd Is this a problematic dependency?

Reply:
@intricate Yeah, I think that'll be a problem. We want `networking` up by `infra` & `lib` so we can combine/reshuffle the "network" functionality among them.